restler/utils/logger.py

```diff
@@ -69,6 +71,40 @@
 LOG_TYPE_REPLAY = 'replay'
 LOG_TYPE_AUTH = 'auth'
 
+class Bug():
+     def __init__(self):
+       
+       self.filepath = None
+       self.reproducible = False
+       self.checkerName = None
```

Comment:
>checkerName [](http://example.com/codeflow?start=12&length=11)

please follow the naming convention of lowercase + underscores (```checker_name```, ```request_order``` below)

Reply:
updated